terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,69 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "filing-maintain" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "psc-verification-api"
+  container_port              = "8080"
+  docker_repo                 = "psc-verification-api"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 160
+  lb_listener_paths           = ["/*"]
+  healthcheck_path            = "/"
+  healthcheck_matcher         = "200-302" # no explicit healthcheck in this service yet, change this when added!
```

Comment:
The Health check matcher cannot be a range as this is used in the container health check used in task container health check.